app/config/constants.js

```diff
@@ -55,6 +55,10 @@ export default {
     KENNISNAME: 'http://kanselarij.vo.data.gift/id/concept/beslissings-resultaat-codes/9f342a88-9485-4a83-87d9-245ed4b504bf',
     INGETROKKEN: 'http://kanselarij.vo.data.gift/id/concept/beslissings-resultaat-codes/453a36e8-6fbd-45d3-b800-ec96e59f273b',
   },
+  DECISION_RESULT_CODE_IDS: {
+    GOEDGEKEURD: '56312c4b-9d2a-4735-b0b1-2ff14bb524fd',
+    KENNISNAME: '9f342a88-9485-4a83-87d9-245ed4b504bf',
+  },
```

Comment:
Sorry, late to the review party, but I think this is a dangerous precedent to end up again in an unstructured config file with a mix of ids and URIs. We only consider URIs to be stable. 

I get that you need IDs to filter on multiple values in mu-cl-resources, but I would configure URIs here and get their IDs in the component where you need them. By using [`store.findRecordByUri`](https://github.com/kanselarij-vlaanderen/frontend-kaleidos/blob/development/app/services/store.js#L29) we ensure only 1 backend call is made per concept so performance impact is minimal.

Reply:
hmm, kinda passed over this but we should try to avoid going back to ID's indeed.
We can filter on uri in the query:
```
 'filter[decision-result-code][:uri:]': [
          CONSTANTS.DECISION_RESULT_CODE_URIS.GOEDGEKEURD,
          CONSTANTS.DECISION_RESULT_CODE_URIS.KENNISNAME,
        ].join(','),
```

Reply:
mu-cl-resources support zo'n "or" niet voor URIs, maar ik push zo een change waar we de IDs gewoon binnenhalen

Reply:
"mu-cl-resources support zo'n "or" niet voor URIs" echt? Ik twijfelde even, had het beter eerst lokaal getest...

Reply:
ja... ik weet dat ook alleen omdat ik het gisteren in rocket chat had gevraagd (heb ook eerst zo'n query proberen te schrijven, maar dat gaat dus niet)
https://chat.semte.ch/channel/mu-semtech?msg=2SQi8azvmuDs3NLie